build.gradle

```diff
@@ -77,7 +77,7 @@ sourceSets {
 
 // Docker publishing
 
-ext.set('imagePrefix', (project.hasProperty('dockerRegistry') ? ( "$dockerRegistry/" - 'http://' - 'https://') : "") + 'datastax')
+ext.set('imagePrefix', (project.hasProperty('dockerRegistry') ? ( "$dockerRegistry/" - 'http://' - 'https://') : "") + 'datastax' + '/cass-config-builder')
```

Comment:
This change was necessary for the publishing to github packages to work.

Reply:
stuff like this is so confusing, can we clean it up? docker images can be broken into
`<registry>/<repository>:<tag>`

the vocabulary confused me for awhile but it's not so bad